src/style.css

```diff
@@ -1,3 +1,72 @@
+:root {
+  --active-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  text-transform: uppercase;
+}
+
+.header {
+  background-color: white;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  position: relative;
+  padding: 0 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+
+  margin: 0;
+  list-style: none;
+  padding-left: 0;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  margin-left: 20px;
+  color: black;
+  text-decoration: none;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  position: relative;
+}
+
+.is-active {
+  margin-left: 0;
+  color: var(--active-color);
+}
+
+.is-active::after {
+  content: '';
+  display: block;
+  width: 100%;
+  height: 4px;
+  background-color: var(--active-color);
+  position: absolute;
+  bottom: 0;
+  border-radius: 8px;
+}
+
+:hover {
```

Comment:
The `:hover` selector is too generic and will apply to all elements. Consider scoping it to specific elements, such as `.nav__link:hover`, to avoid unintended styling across the document.